### ISSUE TITLE
https://github.com/ShikiSuen/SourceHanSansUI no longer available?

### ISSUE THREAD

**Issue by breezewish** [othercat/CJKFontScript#7]

**Comment by ShikiSuen** [comment]
Yes, No longer available. The Origin Han Sans UI is available on CodePlex at this moment. However, CodePlex is shutting down. I will find a time (probably months later) to migrate it back to GitHub.
